src/main/java/ru/itis/controllers/WishListController.java

```diff
@@ -25,7 +25,8 @@ public WishListController(ItemService itemService, WishListService wishListServi
     @GetMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Get a single wish list")
-    public WishList getWishList(@PathVariable("list-id") Long listId, @RequestHeader(name = "AUTH") String token) {
+    public WishList getWishList(@PathVariable("list-id") Long listId, @RequestHeader(name = "Authorization")
```

Comment:
Let me help you
```java
@GetMapping("/lists/{listId}")
public WishList getWishList(@PathVariable Long listId, ...)
```
Now it is shorter by 12 symbols.

Reply:
fixed, thnx

Reply:
I meant other thing, please keep this stuff
```
@PreAuthorize("isAuthenticated()")
@ApiOperation("Get a single wish list")
```

Reply:
fixed